docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,255 @@
+Using Public Clouds as backends
+===============================
+
+Introduction
+------------
+
+As stated in our `GETTING STARTED guide <../GETTING_STARTED/#location-configuration>`__,
+new backends can be added by creating a region with the right endpoint and
+credentials.
+This section of the documentation shows you how to set up our currently
+supported public cloud backends:
+
+- `Amazon S3 <#aws-s3-as-a-backend>`__ ;
+- `Microsoft Azure <#microsoft-azure-as-a-backend>`__ .
+
+AWS S3 as a backend
+-------------------
+
+locationConfig.json
+~~~~~~~~~~~~~~~~~~~
+
+Add a new region, which should describe the AWS S3 hosted bucket you'll be
+writing your data to. There are a few configurable options here:
+
+- :code:`type` : set to :code:`aws_s3` to indicate this region is hosted in AWS S3;
+- :code:`legacyAwsBehavior` : set to :code:`true` to indicate this region should
+  behave like AWS S3 :code:`us-east-1` region, set to :code:`false` to indicate
+  this region should behave like any other AWS S3 region;
+- :code:`bucketName` : set to an *existing bucket* in your AWS S3 Account; this
+  is the bucket that will host all the data written in this region;
+- :code:`awsEndpoint` : set to your bucket's endpoint, usually :code:`s3.amazonaws.com`;
+- :code:`bucketMatch` : set to :code:`true` if you want your object name to be the
+  same in your local bucket and your AWS S3 bucket; set to :code:`false` if you
+  want your object name to be of the form :code:`{{localBucketName}}/{{objectname}}`
+  in your AWS S3 hosted bucket
+- :code:`credentialsProfile` : set to the profile name allowing you to access your
```

Comment:
I don't know if we are, that's probably a question for @LaurenSpiegel ?
I think advocating for using profiles is more scalable, though, as that means they can put all their profiles in one .aws/credentials file and then configure their regions with the different credentials more easily (it's easier to check a human-readble name matches, than randomly generated id/signatures), don't you think?

Reply:
We are allowing both.  In federation, we will be using the flat accessKey, secretKey approach since it is easier than having people create another file in the environment.  So, we should have an example of both.

Reply:
Added the credentials option :)